### DISCUSSION TITLE
Composer install not necessary?

### DISCUSSION THREAD

**Discussion by Aetles** [gregsullivan/_tw#57] [Q&A] [answered]
In the [documentation](https://github.com/automattic/_s) for Underscores they run `composer install` before `npm install`, is that not necessary with _tw or is it presumed that you've already done that?

**Comment by gregsullivan** [comment, marked as answer]
Hi Adrian—Thanks for posting this!

I suppose it depends on your definition of necessary. As the documentation has evolved, it looks like I've removed all mention of running `composer install`, and I don't think that was the right thing to do. That said, it's not necessary for the Tailwind side of things. Composer scripts are only used for two things:

* PHP_CodeSniffer
* Creating POT files via WP-CLI

I'm going to flag this as an omission, though, for when I do my next pass at the documentation.

**Reply by gregsullivan** [reply]
I decided to go ahead and add a paragraph at the end of this page:

https://underscoretw.com/docs/installation/

Please let me know if you think that covers it!

**Reply by Aetles** [reply]
I suspected something it was something along those lines but now I know for sure. The updated documentation makes it very clear. Thank you for the fast feedback (and for _tw!)

**Reply by gregsullivan** [reply]
You're very welcome! Thank you again for this feedback. It wasn't an intentional omission, but rather something that came out of having rewritten and restructured the documentation several times since it was first written.
